### PR TITLE
Deploy charging prerequisites

### PR DESCRIPTION
**Pull request by ArturGajowy**
## Overview
<sup>_What this PR does, and why it's needed_</sup>

### JIRA ticket:
<sup>_Create it if there isn't one already._</sup>

### Notes
<sup>_Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else._</sup>

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

File: casper/src/main/scala/coop/rchain/casper/genesis/Genesis.scala [hunk 1]
```diff
@@ -292,8 +291,8 @@ object Genesis {
         Sync[F].delay {
           keys.foreach { //create files showing the secret key for each public key
             case (sec, pub) =>
-              val sk      = Base16.encode(sec)
-              val pk      = Base16.encode(pub)
+              val sk      = Base16.encode(sec.bytes)
```
Comment (odeac) [inline casper/src/main/scala/coop/rchain/casper/genesis/Genesis.scala]: Not now but at some point later we should hide the bytes and allow only access via some API like fromBase16/toBase16/sign/verify.

**Comment by ArturGajowy** [comment]
Everyone, please feel free to `bors r+` once the 2nd approval lands :)

**Pull request merged**